.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ on:
 
 env:
     fail-fast: true
+    PHPUNIT_FLAGS: "-v"
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+    SYMFONY_REQUIRE: ">=3.4"
```

Comment:
Does it work without flex installed globally?

Reply:
There is a (commented step down below)

This is based on `travis.yml`, but it fails there too, check Travis build: https://github.com/symfony/maker-bundle/pull/699